docs/graphql-api.md

```diff
@@ -139,6 +139,24 @@ _not_ends_with
 
 Please note that some suffixes are only supported for specific types. For example, `Boolean` only supports `"_not", "_in", "_not_in"`.
 
+# 2 Subscriptions 
```

Comment:
Instead of making up new content, let's cherry-pick the existing parts from https://github.com/graphprotocol/specs/tree/master/graphql-api#2-subscriptions and adjust them to match the current state of the implementation.